ramp-engine/ramp_engine/tests/_config.yml

```diff
@@ -5,9 +5,6 @@ ramp:
     event_is_public: true
     data_dir: ramp-board/ramp_deployment/ramp-data/iris
     kit_dir: ramp-board/ramp-engine/ramp_engine/tests/kits
-    submissions_dir: ramp-board/ramp-engine/ramp_engine/tests/kits/iris/submissions
-    predictions_dir: ramp-board/ramp-engine/ramp_engine/tests/kits/iris/predictions
-    logs_dir: ramp-board/ramp-engine/ramp_engine/tests/kits/iris/logs
```

Comment:
The submission dir should be in the ramp section shoudn't it?

Reply:
@rth 
not really. Here are the docs on how the config for AWS should look like:
https://paris-saclay-cds.github.io/ramp-docs/ramp-board/stable/workers.html#event-configuration

The worker gets the submissions from the AWS and needs to know which directory to copy them to on the server:
https://github.com/paris-saclay-cds/ramp-board/blob/fa193ace47a1c14c7d80d0b999704df8f6018923/ramp-engine/ramp_engine/aws/worker.py#L76

Reply:
Ahh, interesting! This does solve an issue I was having in DaskWorker about having a separate local and remote predictions_dir. I should read more on how AWS worker works. But then isn't it incompatible with the [`generate_worker_config`](https://github.com/paris-saclay-cds/ramp-board/blob/fa193ace47a1c14c7d80d0b999704df8f6018923/ramp-utils/ramp_utils/worker.py#L46) util which explicitly loads this information from the `ramp` config section? 

Fine to merge this but, maybe we could rename this file to `_aws_config.yml` or something more specific (not necessarily in this PR) ?

Reply:
Hm. you are right. the submission folder is not a mandatory parameter in the ramp config and if it is not given the default will be used:
https://github.com/paris-saclay-cds/ramp-board/blob/fa193ace47a1c14c7d80d0b999704df8f6018923/ramp-utils/ramp_utils/ramp.py#L92

but this will probably cause problems if the worker receives different submission directory from what is in  ramp_config.

I am not sure what would be the best way out of this

Reply:
Specifying it in both sections (one for remote and one for local)?

Reply:
you mean requiring the user to type the same thing in the ramp_event config in the ramp section and in the worker section?